docker-compose.test.yml

```
@@ -0,0 +1,12 @@
+version: '3.7'
+
+services:
+    sup:
```

The build tests are failing:

> No such service: sut
> starting "sut" service in docker-compose.test.yml (1)

```suggestion
    sut:
```